src/api/controllers/verifyEmail.js

```diff
@@ -8,20 +8,22 @@ import errorMessages from '../../helpers/constants/error.messages';
  */
 export default class {
   /**
-     * @description verification link controller
-     * @param {*} req
-     * @param {*} res
-     * @returns {*} void
-     */
+   * @description verification link controller
+   * @param {*} req
```

Comment:
{*} remember you don't like it

Reply:
This is daniels file
He use to do that mistake

Reply:
I changed a helper and it affected his file.